src/core/layout/qgslayoutitempicture.cpp

```diff
@@ -454,7 +455,7 @@ void QgsLayoutItemPicture::updateMapRotation()
     return;
 
   // take map rotation
-  double rotation = mRotationMap->mapRotation();
+  double rotation = mRotationMap->mapRotation() + mRotationMap->itemRotation();
```

Comment:
```suggestion
  double rotation = mRotationMap->mapRotation() + mRotationMap->rotation();
```

This one is subtle, but check the docs for itemRotation as there's an explanation in those

Reply:
since the only way to access the attribute in the layoutitem was through this I thought it was the only way to get this variable. I forgot to read the doc of the existing variable and only checked the output. My mistake :/

Reply:
Totally understandable.. I missed this one too!   It's a rather poor API, but can't be fixed because the underlying qt class method isn't virtual...